src/app/Library/CrudPanel/Traits/FieldsProtectedMethods.php

```diff
@@ -112,13 +112,15 @@ protected function makeSureFieldHasName($field)
         }
 
         if (is_string($field)) {
-            return ['name' => $field];
+            return ['name' => $this->trimWhitespacesFromName($field)];
```

Comment:
Wouldn't this do the same thing?

```suggestion
            return ['name' => Str::squish($field)];
```

Reply:
If squish doesn't do it (not sure it replaces the spaces in the middle), we could use `Str::replace(' ', '', $field)`